Makefile

```diff
@@ -11,7 +11,7 @@ RELEASE_ROOT = "release"
 RELEASE_GOGS = "release/gogs"
 NOW = $(shell date -u '+%Y%m%d%I%M%S')
 
-.PHONY: build pack release bindata clean
```

Comment:
Oh sorry, I removed bindata to test the build and accidentally committed it... I'll revert that.